src/main/java/frc/trigon/robot/poseestimation/poseestimator/PoseEstimator.java

```diff
@@ -39,10 +46,23 @@ public class PoseEstimator implements AutoCloseable {
     };
     private Rotation2d lastGyroHeading = new Rotation2d();
 
+    /**
+     * Constructs a new PoseEstimator and sets the relativeRobotPoseSource.
```

Comment:
Explain how this constructor differs from the other one, and what happens with the relative robot pose source